### ISSUE TITLE
Timer is not updated with new props (seconds)

### ISSUE THREAD

**Issue by zchauhan** [pughpugh/react-countdown-clock#45]
I guess issue is with `componentDidUpdate` method, if seconds is different in previous and new props then it is selecting previous props not the new one.

```componentDidUpdate: (props) ->
    if props.seconds != @props.seconds
      @_seconds = props.seconds
      @_setupTimer()```

**Comment by pughpugh** [comment]
Agreed, that looks suspect. Thanks, I will take a look.

**Comment by zaidchauhan** [comment]
@pughpugh  PR for the same - https://github.com/pughpugh/react-countdown-clock/pull/46

**Comment by pughpugh** [comment]
Fixed in [2.1.0](https://github.com/pughpugh/react-countdown-clock/tree/2.1.0).
